addon/ng2/blueprints/component-test/files/src/app/components/__name__/__name__.spec.ts

```diff
@@ -15,7 +15,7 @@ import {<%= classifiedModuleName %>} from './<%= dasherizedModuleName %>';
 
 describe('<%= classifiedModuleName %> Component', () => {
 
-  beforeEachProviders(() => []);
+  beforeEachProviders(() => [<%= classifiedModuleName %>]);
```

Comment:
This one should not be necessary even if it is necessary in pipes blueprint. Can you remove?